tests/commands/ssh.spec.ts

```diff
@@ -178,7 +178,7 @@ async function startMockSshServer(): Promise<[Server, number]> {
 	});
 
 	return await new Promise<[Server, number]>((resolve, reject) => {
-		// TODO: remove 'as any' below. According to @types/node v12.20.42, the
+		// TODO: remove 'as any' below. According to @types/node v14.18.36, the
```

Comment:
I tested removing the `as any` with the new version of `@types/node` but the code also fails to complain because of the error in the callback. I just updated the comment to match the node version